src/main.rs

```diff
@@ -439,32 +489,167 @@ impl sqllogictest::AsyncDB for Postgres {
         // and we have to follow the format given by the specific database (pg).
         // For example, postgres will output `t` as true and `f` as false,
         // thus we have to write `t`/`f` in the expected results.
-        let rows = self.client.simple_query(sql).await?;
-        for row in rows {
-            match row {
-                tokio_postgres::SimpleQueryMessage::Row(row) => {
-                    for i in 0..row.len() {
-                        if i != 0 {
+        if !self.extend {
```

Comment:
I'd prefer making it a new engine instead of using `self.extend` to change query mode.